code/game/objects/items/toys.dm

```diff
@@ -809,6 +809,16 @@
 	desc = "An adorable stuffed toy that resembles a lizardperson."
 	icon_state = "plushie_lizard"
 	item_state = "plushie_lizard"
+	var/cooldown = 0
+
+/obj/item/toy/plushie/lizardplushie/attack_self(mob/user)
+	if(!cooldown)
+		playsound(user, 'sound/voice/weh.ogg', 10, 0)
+		visible_message("<span class='danger'>Weh!</span>")
+		cooldown = 1
+		spawn(30) cooldown = 0
```

Comment:
```suggestion
		spawn(30) cooldown = 0
```
come on

Reply:
Stolen from the vox plush code.

Reply:
That doesn't mean it's good like that, at the very least use a varset callback for this

Reply:
Replace this with a varset callback then and make this true/false bools

Reply:
Even better, dont use a timer and use the tg_cooldowns instead

Reply:
Time to refactor all the plushies.

Reply:
> Even better, dont use a timer and use the tg_cooldowns instead

It is done.